plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreDataSource.java

```diff
@@ -931,4 +931,12 @@ public boolean isSupportsEnumTable() {
     public boolean isSupportsReltypeColumn() {
         return supportsReltypeColumn;
     }
+
+    public boolean isPGObject(Object object) {
```

Comment:
What's the point of moving it to PostgreDataSource class? I don't see any reason